src/buyingcatalogue/Chart.yaml

```diff
@@ -29,59 +29,59 @@ dependencies:
   version: ~0.1.0
 - name: bapi-db-deploy
   condition: bapi-db-deploy.enabled
-  version: 1.24.1
+  version: 1.27.0-alpha.122
   repository: "@gpitfuturesdevacr"
 - name: bapi
   condition: bapi.enabled
-  version: 1.24.1
+  version: 1.27.0-alpha.122              
   repository: "@gpitfuturesdevacr"
 - name: dapi
   condition: dapi.enabled
-  version: 1.24.1
+  version: 1.25.0-alpha.14 
   repository: "@gpitfuturesdevacr"
 - name: isapi
   condition: isapi.enabled
-  version: 1.24.4
+  version: 1.29.0-alpha.3 
   repository: "@gpitfuturesdevacr"
 - name: isapi-db-deploy
   condition: isapi-db-deploy.enabled
-  version: 1.24.4
+  version: 1.29.0-alpha.3
   repository: "@gpitfuturesdevacr"
 - name: oapi
   condition: oapi.enabled
-  version: 1.24.4
+  version: 1.29.0-alpha.3
   repository: "@gpitfuturesdevacr"
 - name: ordapi
   condition: ordapi.enabled
-  version: 1.24.1
+  version: 1.27.0-alpha.22 
   repository: "@gpitfuturesdevacr"
 - name: ordapi-db-deploy
   condition: ordapi-db-deploy.enabled
-  version: 1.24.1
+  version: 1.27.0-alpha.22
   repository: "@gpitfuturesdevacr"
 - name: email
   condition: email.enabled
-  version: 1.24.4
+  version: 1.29.0-alpha.3
   repository: "@gpitfuturesdevacr"
 - name: file-loader
   condition: file-loader.enabled
-  version: 1.24.1
+  version: 1.25.0-alpha.14
   repository: "@gpitfuturesdevacr"
 - name: mp
   condition: mp.enabled
-  version: 1.26.1
+  version: 1.28.0-alpha.2
   repository: "@gpitfuturesdevacr"
 - name: pb
   condition: pb.enabled
-  version: 1.24.1
+  version: 1.27.0-alpha.2
   repository: "@gpitfuturesdevacr"
 - name: admin
   condition: admin.enabled
-  version: 1.24.1
+  version: 1.28.0-alpha.2
   repository: "@gpitfuturesdevacr"
 - name: of
   condition: of.enabled
-  version: 1.24.1
+  version: 1.25.0-alpha.740
   repository: "@gpitfuturesdevacr"
 - name: redis
   condition: redis.enabled
```

Comment:
I think that this file should stay unchanged

Reply:
Ticket 7998 raised for this fix as it will not be performed on this version